<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>597</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1124</width>
    <height>597</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1124</width>
    <height>597</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Semilight</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Квадрат</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/pc5dn58Ei.png</normaloff>:/images/pc5dn58Ei.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(214, 214, 214);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>13</y>
      <width>1081</width>
      <height>543</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="8">
      <widget class="QLabel" name="label_images">
       <property name="maximumSize">
        <size>
         <width>581</width>
         <height>541</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/images/18254_900.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_A">
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Точка А:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_xA">
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Ax">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_B">
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Точка B:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_xB">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="lineEdit_Bx">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_yA">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Ay">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_yB">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="lineEdit_By">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_C">
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Точка C:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_xC">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Cx">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_D">
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Точка D:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_xD">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLineEdit" name="lineEdit_Dx">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_yC">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Cy">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_yD">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLineEdit" name="lineEdit_Dy">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLabel" name="label_lenA">
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Длина стороны a:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_result_length_a">
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6" rowspan="2" colspan="2">
      <widget class="QPushButton" name="pushButton_calc">
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);</string>
       </property>
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_perimeter">
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Периметр:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_result_perimeter">
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_area">
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Площадь:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_result_area">
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6" colspan="2">
      <widget class="QPushButton" name="pushButton_clean">
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);</string>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="7">
      <widget class="QPlainTextEdit" name="plainTextEdit_report">
       <property name="maximumSize">
        <size>
         <width>531</width>
         <height>161</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_save_file"/>
    <addaction name="action_open_file"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>О разработчике</string>
   </property>
  </action>
  <action name="action_save_file">
   <property name="text">
    <string>Сохранить данные</string>
   </property>
  </action>
  <action name="action_open_file">
   <property name="text">
    <string>Добавить данные</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
